/**:
  ros__parameters:
    steer_center : 380
    steer_limit : 100
    steer_dir : -1
    speed_center: 0
    speed_limit: 4096
    has_steer : 1
    isDCSteer : 0
    i2cSteer: 96
    i2cThrottle: 64
    k_steer: 2.5
    k_throttle: 0.2

    det_class : "cup"
    det_class1 : "left"
    det_class2 : "right"
    det_class3 : "go"
    det_class4 : "stop"
